src/rust/lib/graph-editor/src/component/visualization/renderer/example/native.rs

```diff
@@ -101,3 +103,80 @@ impl display::Object for BubbleChart {
         &self.display_object.display_object()
     }
 }
+
+
+
+// ===============================
+// === Native RawText Renderer ===
+// ===============================
+
+/// Sample visualization that renders the given data as text. Useful for debugging and testing.
+#[derive(Debug)]
+#[allow(missing_docs)]
+pub struct RawText {
+    scene          : Scene,
+    root_node        : DomSymbol,
+    size               : Cell<Vector2<f32>>,
+    frp                : DataRendererFrp,
+    logger             : Logger,
+}
+
+impl RawText {
+    /// Constructor.
+    pub fn new(scene:&Scene) -> Self {
+        let logger    = Logger::new("bubble");
+        let div       = web::create_div();
+        let root_node = DomSymbol::new(&div);
+        let frp       = default();
+        let size      = Cell::new(Vector2::zero());
+        let scene     = scene.clone_ref();
+
+        // TODO: Fix rotation in DomSymbol.
+        root_node.set_rotation(Vector3::new(180.0_f32.to_radians(), 0.0, 0.0));
```

Comment:
Why are you rotating around the X-Axis :O Is this making a mirror image? This is definitely strange.